json/Side_Text.txt

```diff
@@ -228828,1335 +228828,1335 @@
 		"jp_name": "",
 		"tr_name": "",
 		"jp_text": "オレがきろくを始めて<%br>今日で７２回目になる。",
-		"tr_text": "",
+		"tr_text": "Today will be my 72nd report.",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "1",
 		"jp_name": "",
 		"tr_name": "",
 		"jp_text": "ナンバー：７２<%br>天気：はれ<%br>きろくしゃ：モア<%br>ジェネと、リーダーとの三人で<%br>チームを組んだ。",
-		"tr_text": "",
+		"tr_text": "Report Number: 72<%br>Weather: Sunny<%br>Reporting: More<%br>Leader, Gene and I<%br>formed a team today.",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "2",
 		"jp_name": "",
 		"tr_name": "",
 		"jp_text": "リーダーってのは名前ばっかりで<%br>じっさいはオレが<%br>チームをまとめているようなもんだ。<%br>だって、リーダーは強いけど<%br>ちょっとなんかなぁ……",
-		"tr_text": "",
+		"tr_text": "It isn't official that Leader's in charge,<%br>but when you keep calling someone \"Leader\",<%br>they get the idea sooner or later.<%br>It makes sense, since me and Gene<%br>are pretty outclassed in the power department.",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "3",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "おまけに、ジェネはリーダーよりも<%br>抜けてるんだもんなぁ……！",
-		"tr_text": "",
+		"tr_text": "On top of all that, Gene messes up all the time,<%br>and it makes her look real bad next to Leader!",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "4",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "セラフィさんも、言ってたし。<%br>オレが、ふたりを引っぱらないとな！",
-		"tr_text": "",
+		"tr_text": "Even Seraphy's noticing.<%br>I'm gonna have to cover for her!",
```

Comment:
> セラフィさんも、言ってたし。
Seraphy has even said so before. (in reference to the above)

> オレが、ふたりを引っぱらないとな！
I am going to have to work as hard as two people!

Reply:
How's this?